### PR TITLE
Install a unique bridge network for each VCH install

### LINKED ISSUE #2524: Ensure each VCH is on a unique bridge in CI
Need to create a port group when the testing starts, then clean it up afterwards

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Fixes #2524

### CODE REVIEW

**Comment by anchal-agrawal** [comment]
LGTM

File: tests/resources/Util.robot [hunk 1]
```diff
@@ -83,6 +83,10 @@ Install VIC Appliance To Test Server
     Run Keyword And Ignore Error  Cleanup Datastore On Test Server
     Run Keyword And Ignore Error  Cleanup Dangling Networks On Test Server
     Set Test VCH Name
+    # Set a unique bridge network for each VCH that has a random VLAN ID
+    ${vlan}=  Evaluate  str(random.randint(1, 4093))  modules=random
```
Comment (hmahmood) [inline tests/resources/Util.robot]: We do not want to reuse `${vch-name}` for this?

Comment (hmahmood) [inline tests/resources/Util.robot]: As long as you have a unique name, vc should give you a unique vlan id, so I don't see a need for this.

File: tests/resources/Util.robot [hunk 2]
```diff
@@ -364,6 +369,15 @@ Get VM IP
     ${rc}  ${out}=  Run And Return Rc And Output  govc vm.ip ${vm}
     Should Be Equal As Integers  ${rc}  0
     [Return]  ${out}
+    
+Get VCH Host Name
```
Comment (hmahmood) [inline tests/resources/Util.robot]: Is this used anywhere?

Comment (mhagen-vmware) [inline tests/resources/Util.robot]: It is used in the vmotion test I am about to put a PR in for

File: tests/resources/Util.robot (hunk 1 above)
Comment (mhagen-vmware) [inline tests/resources/Util.robot]: All defaulted to VLAN 0
```Name:                    VCH1
Virtual switch:          vSwitch0
VLAN ID:                 0
Active ports:            0
Allow promiscuous mode:  No
Allow forged transmits:  Yes
Allow MAC changes:       Yes

Name:                    VCH2
Virtual switch:          vSwitch0
VLAN ID:                 0
Active ports:            0
Allow promiscuous mode:  No
Allow forged transmits:  Yes
Allow MAC changes:       Yes

Name:                    VCH3
Virtual switch:          vSwitch0
VLAN ID:                 0
Active ports:            0
Allow promiscuous mode:  No
Allow forged transmits:  Yes
Allow MAC changes:       Yes
```

Comment (mhagen-vmware) [inline tests/resources/Util.robot]: This isn't VC maybe? It is just ESX hosts..

Comment (hmahmood) [inline tests/resources/Util.robot]: Great; that means there still a chance we will run into random networking issues in tests.

**Comment by hmahmood** [comment]
lgtm

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**